Coding/server/lib/HTTPsWebServer/HTTPsWebServer.h

```diff
@@ -105,14 +106,17 @@ class HTTPsWebServer
     static const uint8_t MAX_CLIENTS = 4;
 
     /** Specifies MIME type and which file types should be deployed by web server and */
-    static const String m_servedFileTypes[4][2];
+    static const String m_servedFileTypes[6][2];
```

Comment:
Why not use String? I se no reason to use char* here as only Strings are used throughout the project. If I'd use char arrays for string representation, then I'd have a 3D array!? This is a 6x2 array (6 key/value pairs)

Reply:
Because it is constant and wastes memory for heap management. It is not a 3d array, why?
If you try to keep the key/value relation ... hmm ... maybe a struct is better here.

Reply:
Fixed with bd65c68